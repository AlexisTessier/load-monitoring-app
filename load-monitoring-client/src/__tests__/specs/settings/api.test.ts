import {
  apiOrigin,
  host,
  port,
  protocol,
  sseOrigin
} from '../../../settings/api'

test('protocol', () => {
  expect(typeof protocol).toBe('string')
})

test('host', () => {
  expect(typeof host).toBe('string')
})

test('port', () => {
  expect(typeof port).toBe('string')
})

test('apiOrigin', () => {
  expect(typeof apiOrigin).toBe('string')
})

test('sseOrigin', () => {
  expect(typeof sseOrigin).toBe('string')
})

test('snapshot', () => {
  expect({
    protocol,
    host,
    port,
    apiOrigin,
    sseOrigin
  }).toMatchSnapshot()
})
