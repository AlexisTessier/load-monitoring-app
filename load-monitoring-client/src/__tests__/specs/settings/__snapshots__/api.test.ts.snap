// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
Object {
  "apiOrigin": "http://localhost:3002",
  "host": "localhost",
  "port": "3002",
  "protocol": "http",
  "sseOrigin": "http://localhost:3002/channel",
}
`;
